src/NServiceBus.StructureMap/NServiceBus.StructureMap.csproj

```diff
@@ -1,99 +1,35 @@
-      <HintPath>..\packages\StructureMap.4.4.2\lib\net45\StructureMap.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="StructureMapBuilder.cs" />
-    <Compile Include="ConfigurationExpressionExtensions.cs" />
-    <Compile Include="StructureMapExtensions.cs" />
-    <Compile Include="StructureMapObjectBuilder.cs" />
-    <Compile Include="AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
+    <PackageReference Include="StructureMap" Version="4.4.2" />
+    <PackageReference Include="Janitor.Fody" Version="1.4.1" />
+    <PackageReference Include="NServiceBus" Version="6.0.0" />
+    <PackageReference Include="Obsolete.Fody" Version="4.2.2" />
+    <PackageReference Include="Particular.CodeRules" Version="0.2.0" />
+    <PackageReference Include="NuGetPackager" Version="0.6.3" />
+    <PackageReference Include="Fody" Version="2.1.0" />
+    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" />
     <None Include="..\..\packaging\nuget\NServiceBus.StructureMap.nuspec">
       <Link>NServiceBus.StructureMap.nuspec</Link>
     </None>
-    <None Include="packages.config">
-      <SubType>Designer</SubType>
-    </None>
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="FodyWeavers.xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Analyzer Include="..\packages\Particular.CodeRules.0.1.1\tools\..\analyzers\dotnet\cs\Particular.CodeRules.dll" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Particular.CodeRules.0.1.1\build\Particular.CodeRules.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Particular.CodeRules.0.1.1\build\Particular.CodeRules.props'))" />
-    <Error Condition="!Exists('..\packages\NuGetPackager.0.6.0\build\NuGetPackager.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NuGetPackager.0.6.0\build\NuGetPackager.targets'))" />
-    <Error Condition="!Exists('..\packages\Fody.1.29.4\build\dotnet\Fody.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Fody.1.29.4\build\dotnet\Fody.targets'))" />
-    <Error Condition="!Exists('..\packages\GitVersionTask.3.6.3\build\dotnet\GitVersionTask.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\GitVersionTask.3.6.3\build\dotnet\GitVersionTask.targets'))" />
+  <Target Name="BeforeBuildPackages" BeforeTargets="BuildPackages">
```

Comment:
You can drop this if you instead set `<OutputPath>..\..\binaries\</OutputPath>`. You do need to tweak the nuspec to add the targetframework path, but this is something that should probably be done anyway to prepate for multi-targeting.